app/views/content/funding-and-support/scholarships-and-bursaries/_funding-widget.html.erb

```diff
@@ -21,7 +21,9 @@
     link_title: "Learn more about fees and financial support for non-UK trainee teachers.",
     link_url: "/non-uk-teachers/fees-and-funding-for-non-uk-trainees",
   ) %>
-    <p>The figures on this page apply if you're starting your training <%= value :bursaries_fundingdates %>.</p>
+  </section>
+  <section>
+    <%= render Content::InsetTextComponent.new(text: "The figures on this page apply if you're starting your training $bursaries_fundingdates$. Financial support for courses starting the following year will be announced in autumn 2024.") %>
```

Comment:
@emmaachesongray I wonder if we should put this inset component at the top of the page in this interim period - pretty sure that is what we did last year

and then when the new bursaries are out, we can go back to having the date reference here